### PR TITLE
Modify SCP:SL Dedicated to allow EXILED Modification to run Auto Updater

### PR DESCRIPTION
**Pull request by Thiritin**
I'm proposing this change as it will add an Environment Variable used by the EXILED (a Popular SCP:SL Mods Framework) to run its auto updater. Otherwise, the user could run into an issue with his server not starting.

I know this is normally not used to support modifications. But as this is only an environment variable I will propose the change anyway.

### All Submissions:

* [x] Have you followed the guidelines in our Contributing document?
* [x] Have you checked to ensure there aren't other open [Pull Requests](../../pulls) for the same update/change?
* [x] Did you branch your changes and PR from that branch and not from your master branch?
  * If not, why?:

### Changes to an existing Egg:

1. [x] Have you added an explanation of what your changes do and why you'd like us to include them?
2. [x] Have you tested your Egg changes?

### CODE REVIEW

**Comment by Thiritin** [comment]
image
Egg Modification works and Customers EXILED Modification does complete. Before it errored with a message that the `DOTNET_BUNDLE_EXTRACT_BASE_DIR` is not set.

**Comment by Thiritin** [comment]
I also tested a fresh Server Install just to be sure. Succeeds without error (as expected).
image

**Review by parkervcp — CHANGES_REQUESTED** [review]
this could be added as a variable where the startup doesn't need to be edited and the end user can't see this.

File: steamcmd_servers/scpsl/dedicated/egg-scpsl.json [hunk 1]
```diff
@@ -8,7 +8,7 @@
     "author": "info@goover.de",
     "description": "Egg for SCP: Secret Laboratory Dedicated Linux Server",
     "image": "quay.io\/parkervcp\/pterodactyl-images:debian_mono-5-complete",
-    "startup": ".\/LocalAdmin {{SERVER_PORT}}",
+    "startup": "export DOTNET_BUNDLE_EXTRACT_BASE_DIR=.\/dotnet-bundle && .\/LocalAdmin {{SERVER_PORT}}",
```
[the remark this answers is not in the document]
Comment (Thiritin) [reply steamcmd_servers/scpsl/dedicated/egg-scpsl.json]: 51c8a80

**Review by parkervcp — APPROVED** [review]

**Pull request merged**